docs/user_guide/02_medrecord.md

```diff
@@ -108,6 +108,27 @@ This will expand the MedRecord, adding several new nodes to the graph. However,
 :class: transparent-image
 ```
 
+:::{note}
+Nodes can be added to the MedRecord in a lot of different formats, such as a Pandas Dataframe (as previously shown), but also from a [`NodeTuple`](medmodels.medrecord.types.NodeTuple){target="_blank"}:
+
+```{literalinclude} scripts/02_medrecord_intro.py
+---
+language: python
+lines: 36-40
+---
+```
+
+Or a Polars Dataframe:
+
+```{literalinclude} scripts/02_medrecord_intro.py
+---
+language: python
+lines: 42-51
+---
+```
```

Comment:
maybe reorder this? It's a bit weird to show creation from a dataframe, then tuples and then dataframes again